streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorStateManagerTest.java

```diff
@@ -517,7 +533,8 @@ public void shouldThrowIllegalArgumentExceptionOnRegisterWhenStoreHasAlreadyBeen
             noPartitions,
             false,
             stateDirectory,
-            Collections.emptyMap(),
+            emptyMap(),
+            singletonList(mockKeyValueStore.name()),
```

Comment:
note that stores need not be represented in the `name->changelog` map, but they must be listed in the topological order.